mep/people/forms.py

```diff
@@ -86,6 +86,10 @@ class MemberSearchForm(FacetForm):
         ('name', 'Name A-Z'),
     ]
 
+    range_field_map = {
```

Comment:
Do we need this here? Since the solr range stats are being generated in the view, then probably the view should map solr field to form field where they don't match.

Reply:
I moved this over to the view and added the configuration there as needed.